src/main/scala/org/bblfsh/client/v2/package.scala

```diff
@@ -0,0 +1,5 @@
+package org.bblfsh.client
+
+package object v2 {
```

Comment:
Can you please comment what this is for?

Reply:
This only serves for defining package-wide type aliases - do you think it is still worth documenting that, even if that is a standard way of doing so in Scala?

Will add a doc for the type alias.

Reply:
addressed in 8162713fa88a425510391838a9f51c9bb83c5466

Reply:
If this is conventional (and would thus not surprise someone actually familiar with Scala, unlike me) then I don't think there's any need to document it specially.